src/components/Blocking/ReportDomain.js

```diff
@@ -123,13 +123,18 @@ const styles = StyleSheet.create({
     paddingTop: 8,
     marginBottom: 30
   },
+  btnText: {
+    fontFamily: fonts.inter[500],
+    fontSize: 16,
+    color: COLORS.white2
+  },
   btnSkip: {
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
     flexDirection: 'row',
     justifyContent: 'space-between',
     paddingHorizontal: 17,
     paddingVertical: 18,
-    marginVertical: 22
+    marginBottom: 22
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The button styles have been updated, including changes in font family, font size, color, background color, padding, and margin. Ensure these changes are consistent with the overall design of your application.

```diff
-   btnSkip: {
-    backgroundColor: COLORS.lightgrey,
-     flexDirection: 'row',
-     justifyContent: 'space-between',
-     paddingHorizontal: 17,
-     paddingVertical: 18,
-    marginVertical: 22
-   },
+   btnText: {
+     fontFamily: fonts.inter[500],
+     fontSize: 16,
+     color: COLORS.white2
+   },
+   btnSkip: {
+     backgroundColor: COLORS.gray100,
+     flexDirection: 'row',
+     justifyContent: 'space-between',
+     paddingHorizontal: 17,
+     paddingVertical: 18,
+     marginBottom: 22
+   },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->